scripts/runChainTests.ts

```diff
@@ -0,0 +1,195 @@
+import { ArgumentParser, Namespace } from 'argparse';
+import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
+
+import { config, defaultSasBuildOpts } from './config';
+import { IProcOpts, StatusCode } from './types';
+
+// Stores all the processes
+const procs: { [key: string]: ChildProcessWithoutNullStreams } = {};
+
+// Set the env variable for SAS_SUBSTRATE_WS_URL
+const setWsUrl = (url: string): void => {
+	process.env.SAS_SUBSTRATE_WS_URL = url;
+};
+
+/**
+ * Launch any given process. It accepts an options object.
+ *
+ * {
+ *   proc => the name of the process to be saved in our cache
+ *   resolver => string: If the stdout contains the resolver it will resolve the process
+ *   resolverStartupErr => string: If the stderr contains the resolver it will resolve the process
+ *   args => an array of args to be attached to the `yarn` command.
```

Comment:
```suggestion
 *   proc string => the name of the process to be saved in our cache
 *   resolver string => If the stdout contains the resolver it will resolve the process
 *   resolverStartupErr string => If the stderr contains the resolver it will resolve the process
 *   args string[] => an array of args to be attached to the `yarn` command.
```
Or remove types

Reply:
Yea I agree, good point. Will remove them.